build/config/compiler/BUILD.gn

```diff
@@ -506,7 +506,7 @@ config("rtti_default") {
 }
 
 config("no_exceptions") {
-  cflags = [ "-fno-exceptions" ]
+  cflags = [ "-fexceptions" ] # An ugly way to enable exceptions handling 
```

Comment:
Todo: find a better way

Reply:
Try adding `configs -= [ "//build/config/compiler:exceptions" ]` to the app's build script. That should apply the `exceptions` config (lines 512-514).

Reply:
Exceptions are disabled by default. We should try removing `exceptions_default` or `no_exceptions` (if they exist) and add `exceptions`. Needs some experimentation to get into what we expect, i.e. having only `-fexceptions` set.

Reply:
```
$ gn gen out/debug
ERROR at //BUILD.gn:39:16: Item not found
  configs -= [ "//build/config/compiler:no_exceptions" ]
               ^--------------------------------------
You were trying to remove "//build/config/compiler:no_exceptions"
from the list but it wasn't there.
```
```
$ gn gen out/debug
ERROR at //BUILD.gn:39:16: Item not found
  configs -= [ "//build/config/compiler:exceptions_default" ]
               ^-------------------------------------------
You were trying to remove "//build/config/compiler:exceptions_default"
from the list but it wasn't there.
```

Reply:
Currently, the exception has been enabled by modifying the default on the compiler. 

build/config/compiler/BUILD.gn:
```
config("exceptions_default") {
  configs = [ ":exceptions" ] # Enable exceptions handling 
```

Reply:
`configs += [ "//build/config/compiler:exceptions" ]` did not work?

Reply:
No. It didint work:

```